generic.go

```diff
@@ -190,3 +190,29 @@ func bounds(i int, optionalJ ...int) (int, int) {
 
 	return i, j
 }
+
+func Range[T Integer](i, j T) []T {
+	var l, h T
+
+	if i < j {
+		l = i
+		h = j
+	} else {
+		l = j
+		h = i
+	}
+
+	values := make([]T, h-l+1)
+
+	for k := range values {
+		if i < j {
+			values[k] = l
+			l++
+		} else {
+			values[k] = h
+			h--
+		}
+	}
```

Comment:
```suggestion
		if i < j {
			values[k] = l
			l++
			continue
		}
		values[k] = h
		h--
	}
```

Reply:
In this case, I see no difference between using one or the other, its literally swapping 4 lines for 4 lines. If more people agree I can apply the suggestion

Reply:
IMO it's more a matter of style than anything else. I always mentally read the version with `continue` as "if...then...else" anyway.
There are other ways to break this statement to make the idea clearer. We could define the step outside of the for loop based on the bounds and always add it to the value. E.g. :
```go
if h < l {
    return rangeDesc(h, l) 
}

return rangeAsc
```

WDYGT?

Reply:
Yeah, it's just for style, I could agree with @thefuga, about creating a function but it's only 2 lines.

Reply:
I agree with creating the functions.